assets/javascripts/src/actions.es6

```diff
@@ -46,6 +46,7 @@ export function submitPayment(dispatch) {
             if (response.response.ok) dispatch({ type: PAYMENT_COMPLETE, response: response.json })
             else dispatch({ type: PAYMENT_ERROR, kind: 'card', error: response.json })
         })
+        .then(trackPayment(state.card.amount, state.data.currency.code))
```

Comment:
in fact we should probably create an action for paymentComplete, which does both the GA tracking and redirect, and use that here and with paypal